tests/xs/opacity_Fe_test_gamma_KA4.py

```diff
@@ -0,0 +1,81 @@
+"""test for opacity calculation of metal lines
+
+  - This test calculates Fe opacity from VALD3 line list. (Comparison with petitRADTRANS opacity is shown in examples/comparisons/opacity_Fe_VALD3.ipynb)
+    The calculation of gamma is based on the 4rd equation in p.4 of Kurucz&Avrett1981
+    
+  Note: The input line list needs to be obtained from VALD3 (http://vald.astro.uu.se/). VALD data access is free but requires registration through the Contact form (http://vald.astro.uu.se/~vald/php/vald.php?docpage=contact.html). After the registration, you can login and choose the "Extract Element" mode.
+        For this test, the request form should be filled as:
+            Starting wavelength :    1500
+            Ending wavelength :    100000
+            Element [ + ionization ] :    Fe 1
+            Extraction format :    Long format
+            Retrieve data via :    FTP
+            Linelist configuration :    Default
+            Unit selection:    Energy unit: eV - Medium: vacuum - Wavelength unit: angstrom - VdW syntax: default
+        Please rename the file sent by VALD ([user_name_at_VALD].[request_number_at_VALD].gz) to "vald2600.gz" if you would like to use the code below without editing it.
+   
+"""
+
+import pytest
+import numpy as np
+from exojax.spec import xsection, moldb, atomll
+from exojax.spec.hitran import SijT, doppler_sigma
+import matplotlib.pyplot as plt
+from exojax.utils.constants import m_u
+
+filepath_VALD3 = '/home/tako/work/VALD3/vald2600.gz'
+path_fig = '/home/tako/Dropbox/tmpfig/tmp_211111/'
+
+#-------
+
+out_suffix = '_pytest'
+H_He_HH_VMR = [0.0, 0.16, 0.84] #H, He, H2 #pure[1.0, 0.0, 0.0] #test[0.05, 0.005, 0.1] #Solar[0.0, 0.16, 0.84]
+
+nus = 1e8/np.arange(12200, 11800, -0.01, dtype=np.float64) #wavenumber range for opacity calculation (Covering whole wavelength ranges of both IRD and CARMENES)
+nus4LL = 1e8/np.arange(1e5, 1500.0, -0.01, dtype=np.float64) #wavenumber range for LineList being taken into account (Taking all (except for 1e5–1e6) lines in the line lists (VALD3, Kurucz) into consideration)
+pf_Irwin = False #if True, the partition functions of Irwin1981 is used, otherwise those of Barklem&Collet2016
+
+
+#Read line list
+#$ cp [user_name_at_VALD].[request_number_at_VALD].gz vald2600.gz
+adbFe = moldb.AdbVald(filepath_VALD3, nus4LL, Irwin=pf_Irwin)
+
+Amol=np.float64( adbFe.atomicmass[0] ) #atomic mass [u]
+ionE=np.float64( adbFe.ionE[0] ) #ionization energy [eV]
+nu0=adbFe.nu_lines
+
+#-------
+
+@pytest.mark.parametrize("T", [81, 200, 493, 1215, 2217, 2995, 3250, 4000]) #[81, 110, 148, 200, 270, 365, 493, 666, 900, 1215, 1641, 2000, 2217, 2500, 2750, 2995, 3250, 3500, 3750, 4000]
```

Comment:
In the current version, this test has 40 items and it takes very long time. But it's just a unit test. can you remove the number of items? Probably, only 1 parameter set is enough for the purpose of the unit test?

Reply:
But anyway, it worked! (though I cannot wait for the all items....)

``` sh
shirochan:~/exojax/tests/xs(metalline)>pytest opacity_Fe_test.py
================================================= test session starts ==================================================
platform linux -- Python 3.8.8, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
rootdir: /home/kawahara/exojax
plugins: anyio-2.2.0, asdf-2.8.1
collected 40 items                                                                                                     

opacity_Fe_test.py .....................
```

Reply:
Oh, sorry this was for "opacity_Fe_test.py" not "opacity_Fe_test_gamma_KA4.py" but the same comments for those unit tests....